community/blog/2020-05-New-authorization-and-enhanced-logging-in-WebApi-v1.0.1/README.md

```diff
@@ -0,0 +1,76 @@
+# New authorization & enhanced logging in WebApi v1.0
+
+With the release of version [Arcus WebApi v1.0](https://github.com/arcus-azure/arcus.webapi/releases/tag/v1.0.1), the logging functionality is greatly enhanced for better tracing in your current and future applications.
+
+Come and take a look at what it has to offer!
+
+## JWT authorization
+
+This version has so a [renamed component](https://github.com/arcus-azure/arcus.webapi/issues/149) in the new `Arcus.WebApi.Security` package called: JWT authorization.
```

Comment:
```suggestion
As of v1.0, we have renamed our existing `<name>` into `<>` because [we feel it's broader than just Managed Identity](https://github.com/arcus-azure/arcus.webapi/issues/149).
```